### ISSUE TITLE
XML file updates for bills after a Congress ends

### ISSUE THREAD

**Issue by sibiliab** [usgpo/bill-status#243]
Hi,

Is it possible for bill status xml files from previous Congresses to update once the new Congress starts? For example, here's [HR 1916 from the 117th Congress](https://www.govinfo.gov/bulkdata/BILLSTATUS/117/hr/BILLSTATUS-117hr1916.xml). That Congress would've ended around the end of Dec. 2022. The most recent update date for the xml file is 3/8/2023, which occurred during the 118th Congress. 

It definitely seems like it's possible for bill status xml files to update after the end of a Congress, but want to confirm. If they can be updated, do you know for what period of time that is? Is it any period of time or something like 12 months? Also, can you please define the difference between the updateDate tag on an individual xml file and the [Last Modified Date](https://www.govinfo.gov/bulkdata/BILLSTATUS/117/hr) displayed on the complete list of bill statuses for a Congress? Is the Last Modified Date maybe when the update job was last run, regardless of whether or not the file itself was updated?

Thank you!

**Comment by llaplant** [comment]
Yes, it is possible for Bill Status XML files from previous Congresses to be updated after the start of a new Congress. For the Bill Status collection on GovInfo, GPO pulls data from the [Congress.gov API](https://api.congress.gov/). We periodically refresh the data set based updates to the data at the Library of Congress. Note, the current Congress job runs at 4, 8, and 12 every 4 hours; prior Congresses run daily. The `updateDate` tag within an XML file is provided by the Library of Congress as part of the Congress.gov API response data. The Last Modified Date displayed on GovInfo corresponds to when the Bill Status XML file was processed and published on GovInfo.

**Comment by JoshData** [comment]
Just to give an example- Bills can become enacted after the end of a Congress because the president has ~10 days to sign legislation, and that will be reflected in the XML. As I recall, this is actually fairly common because of the rush to pass legislation right before the end of a Congress. I think I have also seen corrections to cosponsor lists after the end of a Congress.

**Comment by JayDeline** [comment]
It is important in congress to never rush through a bill its important to know all the facts before a decision is reached to many mistakes that happen through rushing and costing money in long run.
